geany.spec.in

```diff
@@ -60,13 +60,6 @@ Requires:  pkgconfig gtk2-devel
 This package contains the header files and pkg-config file needed for building
 Geany plug-ins. You do not need to install this package to use Geany.
 
-%prep
-%setup -q
```

Comment:
Oops, no. I re-added the missing lines.
Will squash the commits before merging.